Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -358,3 +358,9 @@ RetainPtr<NSError> nsErrorFromExceptionDetails(const WebCore::ExceptionDetails&)
 @property (nonatomic, readonly) id <_WKWebViewPrintProvider> _printProvider;
 @end
 #endif
+
+#if HAVE(UIFINDINTERACTION)
+@interface WKWebView (UIFindInteraction_Internal) <UITextSearching, UIFindInteractionDelegate>
+@property (nonatomic, readwrite, setter=_setFindInteractionEnabled:) BOOL _findInteractionEnabled;
```

Comment:
Why is this declaration necessary in the Internal header? (the same declaration is in the Private header)